packages/apollo-collaboration-server/src/features/features.service.ts

```diff
@@ -35,10 +41,31 @@ export class FeaturesService {
     private readonly assemblyModel: Model<AssemblyDocument>,
     @InjectModel(RefSeq.name)
     private readonly refSeqModel: Model<RefSeqDocument>,
-  ) {}
+  ) {
+    changeRegistry.registerChange('LocationEndChange', LocationEndChange) // Do this only once
+    changeRegistry.registerChange('LocationStartChange', LocationStartChange) // Do this only once
+  }
 
   private readonly logger = new Logger(FeaturesService.name)
 
+  /**
+   * Changes End -position in GFF3
+   */
+  async changeEndPos(serializedChange: SerializedChange) {
+    const ChangeType = changeRegistry.getChangeType(serializedChange.typeName)
+    const change = new ChangeType(serializedChange)
+    this.logger.debug(`Requested change: ${JSON.stringify(change)}`)
+    try {
+      await change.apply({
+        typeName: 'Server',
+        featureModel: this.featureModel,
+      })
+    } catch (error) {
+      throw error
+    }
+    return []
+  }
+
```

Comment:
As noted above, this should be part of `submitChange` in `ChangeService` and not in `FeatureService`.